opendatahub-io/odh-dashboard
# Add data-ids where possible

Adds data-ids to buttons checkboxes and radio buttons

## Description
Data-ids are added for:

- User Management:
  - Save changes
- Cluster Settings:
  - PVC Size input
  - Restore Default
  - Culling Unlimited radio
  - Culling Limited radio
  - Hour/Minute Inputs
  - Toleration Checkbox
  - Data gathering checkbox 
  - Save Changes
- BYON:
  - Import image
  - Repository input
  - Name input
  - Description Input
  - Software/Package Tabs
  - Add Software/Package (Outer/Inner)
  - Import
  - Cancel 
  - Same for update image modal
  - Edit 
  - Delete 
- Spawner
  - Spawner Tab 
  -  Images - Radio buttons
  -  Sizes dropdown (Skipped)
  - GPU dropdown (Skipped)
  - Env var Dropdown (Skipped)
  - Envvar var input
  - Envar value input
  - Secret checkbox
  - Start
  - Cancel
- Status
  - Return
  - Stop
- Administration
  - Admin tab  
  - Start server
  - Stop all servers
  - Stop server
- Learning center 


## How Has This Been Tested?
1. Deploy the ODH Dashboard
2. Verify presence of data-id in mentioned elements

## Merge criteria:
